app/policies/linked_profile_policy.rb

```diff
@@ -0,0 +1,25 @@
+class LinkedProfilePolicy < ApplicationPolicy
+  def update?
+    record.user == user || is_admin?
+  end
+  alias_method :create?, :update?
+  alias_method :destroy?, :update?
+
+  class Scope < Scope
+    def resolve
+      return scope.where(private: true) unless user
```

Comment:
This seems backwards.  Limit anonymous users (`unless user`) to a scope where `private: true`?  So they can _only_ see private stuff?